src/app/clusters/webrtc-requestor-server/webrtc-requestor-server.h

```diff
@@ -0,0 +1,169 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
+#include <app/CommandHandlerInterface.h>
+#include <app/ConcreteAttributePath.h>
+#include <app/reporting/reporting.h>
+#include <lib/core/CHIPError.h>
+#include <protocols/interaction_model/StatusCode.h>
+
+#include <string>
+#include <vector>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace WebRTCTransportRequestor {
+
+class WebRTCRequestorDelegate;
+
+using ICEServerStruct     = Structs::ICEServerStruct::DecodableType;
+using WebRTCSessionStruct = Structs::WebRTCSessionStruct::Type;
```

Comment:
Use ICEServerDecodableTypeStruct and WebRTCSessionTypeStruct to avoid namespace collission

Reply:
What collisions would that be avoiding, exactly?